pages/linux/pmount.md

```diff
@@ -1,7 +1,7 @@
 # pmount
 
 > Mount arbitrary hotpluggable devices as a normal user.
-> More information: <https://linux.die.net/man/1/pmount>.
+> More information: <https://www.unix.com/man-page/suse/1/pmount>.
```

Comment:
It seems like `pmount` has an Ubuntu manual page:

```suggestion
> More information: <http://manpages.ubuntu.com/manpages/focal/en/man1/pmount.1.html>.
```

Reply:
FWIW, `pmount` is maintained by Debian folks so it seems appropriate to use their manpage frontend here.